test/NotifierTest.cpp

```diff
@@ -382,4 +453,19 @@ TEST_F(NotifierTest, 2Messages2Receivers_RestartReceivers_ExpectFeedback_VectorM
 
    // Shutdown everything
    Shutdown({senderData, receiver1ThreadData, receiver2ThreadData});
-}
\ No newline at end of file
+}
+
+TEST_F(NotifierTest, 1Message1Receiver_ReceiveData) {
+   TestThreadData senderData("sender");
+   TestThreadData receiver1ThreadData("receiver");
+   EXPECT_FALSE(FileIO::DoesFileExist(notifierQueuePath));
+   EXPECT_FALSE(FileIO::DoesFileExist(handshakeQueuePath));
+   // spawn thread to create notifier and poll the ReceiveData function 
+   SpawnReceiveDataReceiver_WaitForStartup(receiver1ThreadData);
+   // spawn thread to create rifle and fire shot
+   SpawnReceiveDataSender_WaitForStartup(senderData);
+
+   ExchangeNotification(senderData, receiver1ThreadData);
```

Comment:
Can you add to the test that the received strings matches the sent strings?

Currently it only checks that something was received, not that the right things are received.